src/core/viewports/orthographic-viewport.js

```diff
@@ -35,13 +35,13 @@ export default class OrthographicViewport extends Viewport {
     near = 1, // Distance of near clipping plane
     far = 100, // Distance of far clipping plane
     left, // Left bound of the frustum
-    top, // Top bound of the frustum
+    bottom, // Top bound of the frustum
     // automatically calculated
     right = null, // Right bound of the frustum
-    bottom = null // Bottom bound of the frustum
+    top = null // Bottom bound of the frustum
   }) {
     right = Number.isFinite(right) ? right : left + width;
-    bottom = Number.isFinite(bottom) ? bottom : top + height;
+    top = Number.isFinite(top) ? top : bottom + height;
```

Comment:
this change will break existing applications (internal and external), where users specify left, top, width, height as props. in such case, top will be used as it is and bottom will not be derived (undefined). try the following instead for line 44 to make compatible with GL coordinates (bottom-up).

`bottom = Number.isFinite(bottom) ? bottom : top - height;`

Reply:
Already resolved the problem in line 38, this has no back-compatibility issue.

Reply:
with the new code, still has compatibility issue, for example height = 100, top = -50, bottom = top - height = -150.

Reply:
agree, for this, we should document _top_ should be larger than _bottom_ if we go with {left, bottom, width, height}.